.github/workflows/tests.yaml

```diff
@@ -0,0 +1,79 @@
+name: Test Suite
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, windows-latest]
+        php: [ "7.0", "7.1", "7.2", "7.3", "7.4" ]
+        phpunit-suffix: [phpunit]
+        include:
+          - platform: ubuntu-latest
+            php: "5.5"
+            phpunit-suffix: phpunit-php5
+          - platform: ubuntu-latest
+            php: "5.6"
+            phpunit-suffix: phpunit-php5
```

Comment:
We also don't run tests for 5.5 or 5.6 on Windows. The `grpc` extension isn't installing for 5.5, so I just removed them both.